contracts/core/modules/lib/CTokenModuleCoreState.sol

```diff
@@ -0,0 +1,86 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { CommonMath } from "../../../lib/CommonMath.sol";
+import { ERC20Wrapper } from "../../../lib/ERC20Wrapper.sol";
+import { IAddressToAddressWhiteList } from "../../interfaces/IAddressToAddressWhiteList.sol";
+
+
+/**
+ * @title CTokenModuleCoreState
+ * @author Set Protocol
+ *
+ * The CTokenModuleCoreState library maintains Core-related state for modules that interact with cTokens.
+ *
+ */
+contract CTokenModuleCoreState {
+
+    /* ============ State Variables ============ */
+
+    // Address of transferProxy contract
+    address public transferProxy;
```

Comment:
This construct is a bit too rigid. Frankly, we should probably have a `CoreModuleState2` that has the `transferProxy` in it. And we could name this something like `cTokenWhitelisted` that only has the cTokenWhitelist and passes in the whitelist